Google.PowerShell/Common/GcloudCmdlet.cs

```diff
@@ -91,27 +91,28 @@ protected string GetFullPath(string filePath)
             ProviderInfo provider = null;
             string[] result;
 
-            // resolve filepath and get the first one
+            // Resolve filepath.
             try
             {
                 result = GetResolvedProviderPathFromPSPath(filePath, out provider).ToArray();
             }
             catch (ItemNotFoundException itemEx)
             {
-                // in case the file path is not created but we should still return the resolve path
+                // In case the file path is not created, an error will be thrown.
+                // But we should still return the resolved path.
```

Comment:
It should return fully qualified path. I added a check if the item name is rooted then I return it, otherwise I return ```filePath```.